src/node.cc

```diff
@@ -3661,16 +3665,19 @@ static void ParseArgs(int* argc,
     if (debug_options.ParseOption(arg)) {
       // Done, consumed by DebugOptions::ParseOption().
     } else if (strcmp(arg, "--version") == 0 || strcmp(arg, "-v") == 0) {
+      DisallowInNodeopts(argv[0], is_nodeopt, arg);
```

Comment:
Probably worth listing all the options you've excluded in the [first comment](https://github.com/nodejs/node/pull/12028#issue-216918024).